app/views/collections/_document_list.html.erb

```diff
@@ -1,3 +1,5 @@
+<% if documents.empty? %> <h3><%= t('sufia.collection.no_works') %></h3><br>
```

Comment:
I'm not sure why you'd want the `<br>` in there.  If it needs a margin, give the `h3` an id or class attribute and add the margin to the style sheet.